test/unit/config/test__base.py

```diff
@@ -360,4 +360,4 @@ def test_containerization_unsafe_write_setting(tmp_path, runtime, mocker):
         'podman': 1,
     }
 
-    assert rc.env.get('ANSIBLE_UNSAFE_WRITES') == expected[runtime]
+    assert str(rc.env.get('ANSIBLE_UNSAFE_WRITES')) == str(expected[runtime])
```

Comment:
Env vars values are always strings so I wouldn't apply `str()` to the left side of the expression. The only exception is `None` but It feels wrong to stringify it.
Also, since the expected value is for an env var, I'd just expect a string.

Reply:
It's set to a string `'1'` in the code, so it should be a string in the test case. Sorry I didn't set it properly in the test.